README.md

```diff
@@ -39,9 +39,9 @@ module "db" {
   maintenance_window = "Mon:00:00-Mon:03:00"
   backup_window      = "03:00-06:00"
 
-  # Enhanced Monitoring
-  monitoring_interval = "${var.monitoring_interval}
-  monitoring_role_arn = "${var.monitoring_interval == "0" ? "" : aws_iam_role.rds_monitoring.arn}"
+  # Enhanced Monitoring - see example for details on how to create the role
+  # monitoring_interval = "10"
+  # monitoring_role_arn = "aws_iam_role.rds_enhanced_monitoring.arn"
```

Comment:
Write how the end result will have to look like - `monitoring_role_arn = "arn:aws:iam::123456789012:role/RDSEnhancedMonitoring"` and leave it to curious user to figure out how to get the value (maybe he will use data-source to find the one he has already make, maybe he will create resource himself, maybe he will hardcode it, maybe he passes it as variable, ...)

Reply:
Is it worth putting in a link to the relevant AWS page?

Reply:
I think so, a small note is usually a good idea. In general, I would love to hear from real users what do they miss.